eng/pipelines/templates/jobs/pack.yml

```diff
@@ -32,7 +32,8 @@ jobs:
 
   - template: /eng/pipelines/templates/steps/use-rust.yml@self
     parameters:
-      Toolchain: nightly
+      Toolchain: default
```

Comment:
Shouldn't "default" be the parameter default or something?

Reply:
I can see an argument for that.   Given $(System.DefaultWorkingDirectory), `template: /eng/pipelines/templates/steps/use-rust.yml@self` with no parameters should install the appropriate version of rust for that directory